### ISSUE TITLE
LoV not correct

### ISSUE THREAD

**Issue by mkoatwork** [EBISPOT/webulous#5]
Selecting subclasses of "assay screening campaign stage" from BAO includes also the term ""assay screening campaign stage" into the list of selectable terms. This is also true if descendants is selected
![grafik](https://user-images.githubusercontent.com/19821604/27646777-d9b6842c-5c29-11e7-8a1c-92ffd166cd76.png)

**Comment by simonjupp** [comment]
Yes, at the moment the subclasses and descendants is inclusive. Sometimes people want this, sometimes not. We can add an option to the template so people can choose. I'll create a ticket for this.
